test/e2e/vue.test.js

```diff
@@ -1,141 +1,206 @@
-      const vueVersion = vueURL.match(/vue(\d+)/)[1]; // vue2|vue3
-      const vueData = testData[`vue${vueVersion}`];
+    return config;
+  }
 
-      for (const executeScript of ['unspecified', true]) {
-        test(`handles Vue v${vueVersion}.x advanced usage when executeScript is ${executeScript}`, async () => {
-          const docsifyInitConfig = {
-            markdown: {
-              homepage: vueData.markdown,
-            },
-            scriptURLs: vueURL,
-          };
+  // Tests
+  // ---------------------------------------------------------------------------
+  for (const vueURL of vueURLs) {
+    const vueVersion = Number(vueURL.match(/vue(\d+)/)[1]); // 2|3
 
-          if (executeScript !== 'unspecified') {
-            docsifyInitConfig.config = {
-              executeScript,
-            };
-          }
+    describe(`Vue v${vueVersion}`, function() {
+      for (const executeScript of [true, undefined]) {
+        test(`renders content when executeScript is ${executeScript}`, async () => {
+          const docsifyInitConfig = getSharedConfig();
+
+          docsifyInitConfig.config.executeScript = executeScript;
+          docsifyInitConfig.scriptURLs = vueURL;
 
           await docsifyInit(docsifyInitConfig);
 
-          await expect(page).toEqualText('#test span', '0');
-          await page.click('#test button');
-          await expect(page).toEqualText('#test span', '1');
+          // Static
+          await expect(page).toEqualText('#vuefor', '12345');
+          await expect(page).toEqualText('#vuecomponent', '0');
+          await expect(page).toEqualText(
+            '#vueglobaloptions p',
+            'vueglobaloptions'
+          );
+          await expect(page).toEqualText('#vueglobaloptions span', '0');
+          await expect(page).toEqualText('#vuemounts p', 'vuemounts');
+          await expect(page).toEqualText('#vuemounts span', '0');
+          await expect(page).toEqualText('#vuescript p', 'vuescript');
+          await expect(page).toEqualText('#vuescript span', '0');
+
+          // Reactive
+          await page.click('#vuecomponent');
+          await expect(page).toEqualText('#vuecomponent', '1');
+          await page.click('#vueglobaloptions button');
+          await expect(page).toEqualText('#vueglobaloptions span', '1');
+          await page.click('#vuemounts button');
+          await expect(page).toEqualText('#vuemounts span', '1');
+          await page.click('#vuescript button');
+          await expect(page).toEqualText('#vuescript span', '1');
         });
       }
 
-      test(`handles Vue v${vueVersion}.x advanced usage when executeScript is false`, async () => {
-        const docsifyInitConfig = {
-          config: {
-            executeScript: false,
-          },
-          markdown: {
-            homepage: vueData.markdown,
-          },
-          scriptURLs: vueURL,
-        };
+      test(`ignores content when Vue is not present`, async () => {
```

Comment:
Perhaps this same exact test should run for the case when `Vue` is present, but `executeScript` is explicitly `false`, to ensure Vue doesn't render in that case.

Reply:
Oh wait, nevermind, it render Vue regardless of `executeScript`. That's better.